generation.py

```diff
@@ -0,0 +1,13 @@
+from random import randint
+
+def generate_matrices():
```

Comment:
why it generates 2 matrices of the same dimensions size: AxB and BxY ? 
it is better to:
- specify as input parameters Height and Width for function;
- generate only one matrix (of specified size Height and Width) in function and return it;
- control for matrix multiplication rule AxB BxC (one dimension of each matrix is equal) outside this function;